docs/use/dapp-staking/for-stakers/index.md

```diff
@@ -0,0 +1,109 @@
+
+No staking rewards are generated during the Voting Period but if users vote and stake on dApps during this period, they will become eligible for the **Bonus Reward**;
+
+**Build&Earn Period** is the period when stakers and dApps start earning rewards;
+Users can still stake tokens during the Build&Earn Period but won’t be eligible for the  Bonus Reward;
+
+At the end of a **Build&Earn Period**, the current period ends. A new period begins, dApp Staking  is reset and all tokens are `unstaked` from dApp(s) but remain `locked`. A new **Voting Period** starts.
+
+As an user, you need to take the following parameters into consideration before using dApp Staking:
+
+### Parameters:
+
+|  | Shibuya | Shiden | Astar |
+| --- | --- | --- | --- |
+| Eras Per Cycle | 28 (~7days) | TBD | TBD |
+| Eras Per Voting Period | 8 (~48hours) | TBD | TBD |
+| Eras Per Build&Earn Period | 20 (~120hours) | TBD |TBD  |
+| Blocks Per Era | 1800 (~6hours) | TBD | TBD |
+
+
+### Basic Staking Rewards:
+
+dApp Staking rewards are generated only if tokens have been staked on a dApp for at least 1 full `Era` and distributed on the following `Era` (Era+1);
+
+:::tip
+
+For stakers, if you stake today, you will start earning tomorrow (next era), and the rewards will be distributed the day after tomorrow.
+
+For dApps If you get into a tier today, you will start earning rewards tomorrow (next era) and the rewards will be distributed the day after tomorrow.
+
+:::
+
+For stakers, the APR is the same for all stakers, regardless of which dApp they stake on;
+For dApps, staking rewards depends on the Tier system, dApps go into rewards tier based on the amount of tokens staked by users.
+
+Rewards must be claimed before voting for a new dApp or during the new **Voting Period**.
+*(If using the Astar Portal, the interface will automatically claim the rewards when voting)*
+
+If a project is **unregistered** from dApp Staking, staked tokens on that project will still receive **basic rewards** as long as the tokens remain staked.
+
+:::warning
+
+ There is no rewards during the Unbonding period when unstaking from dApp Staking;
+
+ :::
+
+### Bonus Staking Rewards:
+
+If an user stake on dApp(s) during the **Voting Period** and keep the same staked amount or higher on a dApp through the whole **Build Period**, he is eligible for a the **Bonus Reward** for this dApp.
+
+:::info
+
+**Example**
+
+- User stake 1500 ASTR on **dApp A** and 1000 ASTR on **dApp B** during the `Voting` Period.
+- During the `Build&Earn` Period, user move 500 ASTR from **dApp B** to **dApp A**;
+- He now has 2000 ASTR on **dApp A** and 5000 ASTR on **dApp B**, so he's still eligible for the `Bonus reward` for dApp A but no longer for **dApp B** because the tokens staked on **dApp** B are less than those staked at the end of the `Voting period` (500 < 1000 ASTR).
```

Comment:
```suggestion
- He now has 2000 ASTR on **dApp A** and 500 ASTR on **dApp B**, so he's still eligible for the `Bonus reward` for dApp A but no longer for **dApp B** because the tokens staked on **dApp** B are less than those staked at the end of the `Voting period` (500 < 1000 ASTR).
```